src/Orchard.Web/Modules/Orchard.Autoroute/ImportExport/HomeAliasImportHandler.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Linq;
+using System.Web.Routing;
+using Orchard.Autoroute.Services;
+using Orchard.ContentManagement;
+using Orchard.Recipes.Models;
+using Orchard.Recipes.Services;
+
+namespace Orchard.Autoroute.ImportExport {
+    public class HomeAliasImportHandler : Component, IRecipeHandler {
+        private readonly IHomeAliasService _homeAliasService;
+        private readonly IContentManager _contentManager;
+
+        public HomeAliasImportHandler(IHomeAliasService homeAliasService, IContentManager contentManager) {
+            _homeAliasService = homeAliasService;
+            _contentManager = contentManager;
+        }
+
+        public void ExecuteRecipeStep(RecipeContext recipeContext) {
+            if (!String.Equals(recipeContext.RecipeStep.Name, "HomeAlias", StringComparison.OrdinalIgnoreCase)) {
```

Comment:
Do we have a coding convention guide somewhere that we can update when there is consensus?